components/brave_wallet/browser/keyring_controller.cc

```diff
@@ -337,6 +368,15 @@ HDKeyring* KeyringController::ResumeDefaultKeyring(
   if (account_no)
     default_keyring_->AddAccounts(account_no);
 
+  // TODO(bbondy):
+  // We can remove this some months after the initial wallet launch
+  // We didn't store account address in meta pref originally.
+  for (size_t i = 0; i < account_no; i++) {
```

Comment:
++i

Reply:
Fixed and rebased in 10bd683f95363d743b2fb72a09278f0b5c3bb92e